graphx/src/main/scala/org/apache/spark/graphx/GraphOps.scala

```diff
@@ -362,12 +362,14 @@ class GraphOps[VD: ClassTag, ED: ClassTag](graph: Graph[VD, ED]) extends Seriali
   def pregel[A: ClassTag](
       initialMsg: A,
       maxIterations: Int = Int.MaxValue,
+      checkpointInterval: Int = 25,
```

Comment:
25 is the value in my test. I checked this value in LDA/ALS, looks like the default value is 10, change it to 10 ?